homeassistant/components/prusalink/strings.json

```diff
@@ -25,6 +25,40 @@
           "pausing": "Pausing",
           "printing": "Printing"
         }
+      },
+      "heatbed_temperature": {
+        "name": "Heatbed temperature"
+      },
+      "nozzle_temperature": {
+        "name": "Nozzle temperature"
+      },
+      "progress": {
+        "name": "Progress"
+      },
+      "filename": {
+        "name": "Filename"
+      },
+      "print_start": {
+        "name": "Print start"
+      },
+      "print_finish": {
+        "name": "Print finish"
+      }
+    },
+    "button": {
+      "cancel_job": {
+        "name": "Cancel job"
+      },
+      "pause_job": {
+        "name": "Pause job"
+      },
+      "resume_job": {
+        "name": "Resume job"
+      }
+    },
+    "camera": {
+      "job_preview": {
+        "name": "Preview"
```

Comment:
You need to adjust the tests accordingly => `camera.mock_title_job_preview` versus `camera.mock_title_preview`